python/graphstorm/model/embed.py

```diff
@@ -208,17 +209,15 @@ def __init__(self,
 
         self.activation = activation
 
-        # NCCL backend is not supported for utilizing learnable embeddings on nodes. It has a
-        # dependency on distDGL (PR https://github.com/dmlc/dgl/pull/5929).
-        # TODO (Israt): Add NCCL support to distDGL.
-        if is_distributed() and get_backend() == "nccl":
-            if self.use_node_embeddings:
-                raise NotImplementedError('NCCL backend is not supported for utilizing \
-                    node embeddings. Please use gloo backend.')
-            for ntype in g.ntypes:
-                if not feat_size[ntype]:
-                    raise NotImplementedError('NCCL backend is not supported for utilizing \
-                        learnable embeddings on featureless nodes. Please use gloo backend.')
+        if dgl.__version__ <= "1.1.2":
```

Comment:
can you merge the if condition with the next if condition?

Reply:
Done